test/common.bash

```diff
@@ -113,7 +113,10 @@ prerequisites_ok () {
 
 # Wrapper for Bats run() to work around Bats bug #89 by saving/restoring $IFS.
 # See issues #552 and #555 and https://stackoverflow.com/a/32425874.
-eval bats_"$(declare -f run)"
+# shellcheck disable=SC2091
+if $(declare -f run &> /dev/null); then
```

Comment:
I think here you want to test with `if ( type -t run > /dev/null 2>&1 )` rather than `declare`: https://stackoverflow.com/a/85903 (see first comment).